### DISCUSSION TITLE
Import sprites from anywhere

### DISCUSSION THREAD

**Discussion by Chris-Mingay** [nesbox/TIC-80#1525] [Q&A] [answered]
Hi all, I have been drawing my sprites in Aseprite and importing them. However I seem to have forgotten how to do this with the `import sprites [FILE]` command.

My project is `C:\TIC80\tic80golf\tic80golf.lua`  and there is a `sprites.gif` file in the same directory.

I am editing my project using an external editor (VS Code). I am using the itch based Pro version of TIC80.

From within TIC80 it seems I cannot `cd` to the above directory to run the import command, it seems I am stuck within the `AppData\Roaming\itch\apps\tic80` directory and cannot cd above it. Am I doing something wrong here?

Thanks

**Comment by nesbox** [comment, marked as answer]
Pls, use `tic80 --fs .` command line parameter to use the current directory as a working folder.
Another thing, pls use PNG format for importing sprites in 0.90

**Reply by Chris-Mingay** [reply]
Nice one thank you!
